runtime/compiler/trj9/optimizer/JProfilingValue.cpp

```diff
@@ -264,34 +264,56 @@ void
 TR_JProfilingValue::lowerCalls()
    {
    TR::TreeTop *cursor = comp()->getStartTree();
+   int32_t numberOfNodes = comp()->getFlowGraph()->getNextNodeNumber();
+   TR_BitVector *backwardAnalyzedListOfAddressNodes = new (comp()->trStackMemory())TR_BitVector();
```

Comment:
please add a space between the argument to new and the TR_BitVector for code style.

Reply:
Can we please get a more helpful variable name for the BitVector? It should describe what the bit vector holds, not how it was generated. Some suggestions could include: evaluatedAddressNodes, addressNodesToCheck, ....